### PR TITLE
Develop/fixes/dlsv2 609 issue with the mismatch of the field names in mobile view compared to desktop view when viewing proficiency on the self assessment page

### PR DESCRIPTION
**Pull request by OluwatobiAwe**
### JIRA link
https://hee-dls.atlassian.net/browse/DLSV2-609

### Description
Responsive header text updated to reflect the same content as desktop view. 
The **Action** column is hidden on Desktop View, but visible on the mobile view to aid usability.

### Screenshots
DesktopCapture
MobileCapture

-----
### Developer checks
(Leave tasks unticked if they haven't been appropriate for your ticket.)

I have:
- [x] Run the formatter and made sure there are no IDE errors (see [info on Text Editor settings](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546185813/DLS+Dev+Process) to avoid whitespace changes)
- [ ] Written tests for the changes (accessibility tests, unit tests for controller, data services, services, view models, etc)
- [x] Manually tested my work with and without JavaScript
- [x] Tested any Views or partials created or changed with [Wave Chrome plugin](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh/related) and addressed any valid accessibility issues
- [ ] Updated/added documentation in [Confluence](https://hee-tis.atlassian.net/wiki/spaces/TP/pages/3546939432/DLS+Code) and/or [GitHub Readme](https://github.com/TechnologyEnhancedLearning/DLSV2/blob/master/README.md). List of documentation links added/changed:
  - [doc_1_here](link_1_here)
- [x] Updated my Jira ticket with information about other parts of the system that were touched as part of the MR and have to be sanity tested to ensure nothing’s broken
- [x] Scanned over my own MR to ensure everything is as expected and it looks right in the browser

### CODE REVIEW

**Review by kevwhitt-hee — CHANGES_REQUESTED** [review]
This looks good, Oluwatobi. Could we also look to fix the issue with the alignment of the Competency item:
image
It seems to switch to right aligned when the text is too long. Could we move it to the left (or work out why it's moving)?

**Comment by OluwatobiAwe** [comment]
The nhsuk base CSS stylesheet targetting tables in mobile views, aligns content to the right. I then added the style to the self-accessment style sheet to reflect the expected behavior only on that page. I did this by adding the text-align property to left.

MobileViewUpdate

File: DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_OverviewTable.cshtml [hunk 1]
```diff
@@ -1,82 +1,82 @@
...
+      </th>
+    </tr>
+  </thead>
+
+  @foreach (var competency in Model)
+  {
+    <tbody class="nhsuk-table__body row-outer" id="comp-@competency.RowNo">
+      <tr class="nhsuk-table__row first-row">
         <td rowspan="@competency.AssessmentQuestions.Count()" class="nhsuk-table__cell">
+          <span class="nhsuk-table-responsive__heading">@competency.Vocabulary </span>
```
Comment (kevwhitt-hee) [inline DigitalLearningSolutions.Web/Views/LearningPortal/SelfAssessments/_OverviewTable.cshtml:34]: I've moved this outside of the details expander and this seems to have fixed the alignment and made it consistent with the other items in the summary without CSS changes.

**Review by kevwhitt-hee — COMMENTED** [review]
I could see a way to fix the alignment problem without the CSS change. This works better with the flags that aren't visible on your screenshot (none of the competencies are flagged in the DB).

**Pull request merged**